### PR TITLE
Manual test checklist

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
Before this we used to add the checklist for local test on different OSs manually to the pull request comments. From now on after this pull request, that checklist will be part of the pull request template.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- OSs local check added to pull request template
```
Comment (sepandhaghighi) [inline CHANGELOG.md:9]: - OSs local checklist added to pull request template

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -3,3 +3,8 @@
 #### What does this implement/fix? Explain your changes.
 
 #### Any other comments?
+
+## Local tests on OSs
```
Comment (sepandhaghighi) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: I would like to suggest a solution based on the currently available facilities.

- [ ] MacOS
    - [ ] Sonoma
- [ ] Windows
    - [ ] Windows 11
- [ ] Linux
    - [ ] Ubuntu 20.04

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Please take a look at my comments 🔥

**Comment by sadrasabouri** [comment]
Kindly please re-review @sepandhaghighi

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**